app/main.py

```diff
@@ -14,4 +14,5 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+
+    phrase.lower().count(letter.lower())
```

Comment:
The result of the count operation is not being returned. You should return the count of occurrences to fulfill the function's purpose.